### PR TITLE
Fixed configuration settings not being applied.

### PR DESCRIPTION
**Pull request by SAED2906**
Changed data.name -> child.name as the data.name points to the 2DColouredRect instead of its parent who's name is actually targeted for the configuration settings.

### CODE REVIEW

**Comment by SAED2906** [comment]
Oh and added CRT/VHS shader, I meant to make two separate PRs but they merged. One PR is fixing the loading of configurations and the other is the addition of CRT/VCR

File: addons/post_processing/node/post_process.gd [hunk 1]
```diff
@@ -10,7 +10,7 @@ func _update_shaders() -> void:
 	for child in get_children():
 		var data : ColorRect = child.get_child(0)
 		if data:
-			_update_shader_parameters(data.name, data.material)
```
Comment (SAED2906) [inline addons/post_processing/node/post_process.gd:13]: This was the reason the configs set by users wasn't being used.

**Comment by KorinDev** [comment]
Thank's dude!
I sadly had no time to fix it this week due to personal life, but im back today.
il merge right away and update the Godot Asset Library plugin!

**Comment by SAED2906** [comment]
Awesome, thank you (:

**Pull request merged**